### DISCUSSION TITLE
Issues reproduction capabilities - providing a template Directus minimal project client side + public tests endpoints

### DISCUSSION THREAD

**Discussion by zguig52** [directus/directus#21953] [Feature Requests]
### Summary

To spare time to reporters as maintainers, it would be nice if Directus could provide additional tools:
- minimal directus reproduction project code (stackblitz or codesandbox for example)
- public test directus instance API access to allow the reproduction code to run (static tokens and some basic collections with different kind of permissions for example)

### Basic Example

https://stackblitz.com/edit/github-qytiun-sesbl7?file=components%2FWebSocket.vue

### Motivation

I just opened an issue related to SDK and was wanting to provide quick repro code. I was inspired by Nuxt issue template which I found great and spares lot of time to reporters as maintainers.

Hereafter the interesting issue template part:
```
Please use a template below to create a minimal reproduction
👉 https://stackblitz.com/github/nuxt/starter/tree/v3
👉 https://codesandbox.io/s/github/nuxt/starter/tree/v3
```

I created a repro based on Nuxt but at the end I was just missing a public test API available to allow the code to run and produce the errors inside the browser, without having to have to clone the code locally and use a local Directus instance.

### Detailed Design

N/A - tools:
- github repo with starter minimal code
- directus cloud project dedicated to tests

Example here (new bug):
https://github.com/nuxt/nuxt/issues/new/choose

### Must Have:
- github repo with starter minimal code
- directus cloud project dedicated to tests

### Should Have:
- one static token and one collection

### Could Have:
- different tokens and collections with different permissions

### Drawbacks

Don't see why not :)

### Alternatives

More time on issues processing. More discussions to understand the bug and needed time to implement it in dev env for maintainers.

### Adoption Strategy

No adoption issue, you would only have to update the issue submission template to provide advices on it.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
